src/main/java/uk/gov/pay/adminusers/expungeandarchive/service/ExpungeAndArchiveHistoricalDataService.java

```diff
@@ -105,6 +112,9 @@ private int archiveServices() {
 
                 serviceDao.merge(serviceEntity);
                 detachUsers(serviceEntity);
+                
+                connectorTaskQueue.addTaskToQueue(
+                        new ConnectorTask(new ServiceArchivedTaskData(serviceEntity.getExternalId()), "service_archived"));
```

Comment:
Not sure if we need a flag to enable this bit but I think it's ok to enable it now.